src/python/doc/wasserstein_distance_sum.inc

```diff
@@ -3,11 +3,11 @@
 
    +-----------------------------------------------------------------+----------------------------------------------------------------------+------------------------------------------------------------------+
    | .. figure::                                                     | The q-Wasserstein distance measures the similarity between two       | :Author: Theo Lacombe                                            |
-   |      ../../doc/Bottleneck_distance/perturb_pd.png               | persistence diagrams. It's the minimum value c that can be achieved  |                                                                  |
-   |      :figclass: align-center                                    | by a perfect matching between the points of the two diagrams (+ all  | :Since: GUDHI 3.1.0                                              |
-   |                                                                 | diagonal points), where the value of a matching is defined as the    |                                                                  |
-   |      Wasserstein distance is the q-th root of the sum of the    | q-th root of the sum of all edge lengths to the power q. Edge lengths| :License: MIT                                                    |
-   |      edge lengths to the power q.                               | are measured in norm p, for :math:`1 \leq p \leq \infty`.            |                                                                  |
+   |      ../../doc/Bottleneck_distance/perturb_pd.png               | persistence diagrams using the sum of all edges lengths (instead of  |                                                                  |
+   |      :figclass: align-center                                    | the maximum). It allows to define sophisticated objects such as      | :Introduced in: GUDHI 3.1.0                                      |
+   |                                                                 | barycenters of a family of persistence diagrams.                     |                                                                  |
+   |      Wasserstein distance is the q-th root of the sum of the    |                                                                      | :Copyright: MIT                                                  |
```

Comment:
The legend of the figure seems redundant, and takes space below the figure. If you want to keep the text, I think it would look better in the middle, with the rest of the text.

Reply:
I removed the text. I guess the doc is explicit enough.